edx/edx-enterprise
# [ENT-227] Restore Django 1.10 support

**Description:** Add back support for Django 1.10, which was dropped on #44

**Sandbox link**: https://pr15273.sandbox.opencraft.hosting

**JIRA:** https://openedx.atlassian.net/browse/ENT-227

**Dependencies:** 

**Merge deadline:** 

**Installation instructions:**

**Testing instructions:**

Sandbox

1. Log into the edx-platform sandbox as one of default users.
2. Navigate around
3. Verify everything works as normal

Unit tests

Simply run `tox`.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.~ Had to manually set the new requirements, because new pylint,pycodestyle,pydocstyle versions give **way** to many errors to fix in this PR.
- [ ] ~Documentation updated (not only docstrings)~ N/A
- [ ] ~Translations are updated~ N/A

**Post merge:**

**Author concerns:** Pylint is unpinned ATM. This caused an **incredible** amount of errors when running the quality checks, due to the updated requirements files. I still have to fix them before this PR can be merged.
